runtime/shiden/src/lib.rs

```diff
@@ -1116,34 +1111,18 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
-parameter_types! {
-    pub const DappStakingMigrationName: &'static str = "DappStakingMigration";
-}
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
 pub type Migrations = (
+    // [contracts fix] Part of shiden-121 (added after v5.33.0 release)
+    astar_primitives::migrations::ForceContractsVersion<Runtime, 14>,
+    pallet_contracts::Migration<Runtime>,
```

Comment:
If I understood correctly, this will downgrade storage version by 1, to ensure we bump it back to the correct one with the new migration, right?

Might be an overkill, but I'd add another dummy struct here that checks in try-runtime hooks that version was bumped back up to the expected one. Just a suggestion.

Reply:
Yes, that's correct.
I'll add that.